src/test/test-random-util.c

```diff
@@ -36,10 +34,11 @@ static void test_pseudo_random_bytes(void) {
 }
 
 static void test_rdrand(void) {
-        int r, i;
+        log_info("/* %s */", __func__);
 
-        for (i = 0; i < 10; i++) {
+        for (unsigned i = 0; i < 10; i++) {
                 unsigned long x = 0;
+                int r;
```

Comment:
I don't mind either way, but other reviewers prefer to always have `r` declared at the top of the function :-)